examples/generalized_linear_models/GLM-ordinal-regression.ipynb

```diff
@@ -375,8 +375,8 @@
     "        pt.concatenate(\n",
```

Comment:
<p>Could you also add the t to "Ben Vincen"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>